frameworks/g-cloud-12/questions/declaration/servicesHaveOrSupportCloudHostingCloudSoftware.yml

```diff
@@ -9,10 +9,9 @@ question: >
 type: radios
 options:
   - label: "Yes"
-  - label: "No"
   - label: "My organisation isn't submitting cloud hosting (lot 1) or cloud software (lot 2) services"
-
-assessment:
-  passIfIn:
-    - "Yes"
-    - "My organisation isn't submitting cloud hosting (lot 1) or cloud software (lot 2) services"
```

Comment:
I feel like we should keep the `assessment` flag in these question, to make sure the questions appear in the pass/fail assessment schema. They are a required field, after all.

```
assessment:
  required: True
```

Reply:
👍